### ISSUE TITLE
Orders table: Add buttons to navigate to the 1st/last page into the pagination + show total Qty of orders

### ISSUE THREAD

**Issue by elena-zh** [gnosis/gp-ui#698]
Related to https://github.com/gnosis/gp-ui/issues/694

Currently, there are buttons that only flip pages one by one + no total Qty of orders
![image.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/e568cb37-427a-45e3-85f0-d794d498d575)

It would be nice to add buttons (like double arrows) that will navigate to the 1st/last pages, and show total Qty of orders for an account
Example: 
![image.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/6c173677-7b46-47e9-8d97-9a1d7223b04d)

**Comment by alfetopito** [comment]
As I believe @henrypalacios mentioned, we don't know how many there are until we reach the end.
That requires a change in the backend API and it's not high priority at the moment.

For now, I say leave as is, potentially adding only the option to go to the first page.

**Comment by elena-zh** [comment]
@alfetopito , agree. That's why I created this issue as a separate task as a future enhancement. 
For now, pagination looks good to me.

**Comment by elena-zh** [comment]
Pagination still needs to be improved. 
Latest discussion: 
https://altoros.slack.com/archives/CSGLUC1BN/p1640094437031700
https://gnosisinc.slack.com/archives/CSGLUC1BN/p1640094437031700
